packages/ui-react/src/containers/ShelfList/ShelfList.module.scss

```diff
@@ -24,7 +24,7 @@
   }
 
   @include responsive.tablet-only() {
-    padding: 0 calc(#{variables.$base-spacing} * 2);
+    padding: 24px calc(#{variables.$base-spacing} * 2);
```

Comment:
Why not use base-spacing here as well?

Reply:
Because most of the paddings in this file are not derived from base-spacing 😄